docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -289,23 +151,3 @@ is `host.name: "Windows-ArsenalFC"`.
 
 NOTE: Refer to <<timelines-ui>> for information on creating Timelines and Timeline
 templates. For information on how to add Timeline templates to rules, refer to <<rules-ui-create>>.
-
-[float]
-[[add-exception-from-alerts]]
-=== Add rule exceptions
-
-You can add exceptions to the rule that generated the alert directly from the
-Alerts table. Exceptions prevent a rule from generating alerts even when its
-criteria are met.
-
-To add an exception, click the *More actions* menu (*...*) in the Alerts table, then select
-*Add exception*. Alternatively, select *Take action* -> *Add rule exception* in the Alert details flyout.
-
-For information about exceptions and how to use them, refer to
-<<detections-ui-exceptions>>.
-
-[float]
-[[alerts-analyze-events]]
-=== Visually analyze process relationships
-
-For process events that are detected by {elastic-endpoint}, you can open a visual mapping to view a hierarchal timeline of when these events occurred. For more information, refer to <<visual-event-analyzer>>.
```

Comment:
Removed this subsection, since it's just a sentence and a link. Added a link to the "Take actions" section instead.